applications/dashboard/views/modules/me.php

```diff
@@ -31,10 +31,10 @@
       echo '<div class="MeMenu">';
          // Notifications
          $CountNotifications = $User->CountNotifications;
-         $CNotifications = is_numeric($CountNotifications) && $CountNotifications > 0 ? '<span class="Alert">'.$CountNotifications.'</span>' : '';
+         $CNotifications = is_numeric($CountNotifications) && $CountNotifications > 0 ? '<span class="Alert NotificationsAlert">'.$CountNotifications.'</span>' : '';
 
          echo '<span class="ToggleFlyout" rel="/profile/notificationspopin">';
-         echo Anchor(Sprite('SpNotifications', 'Sprite Sprite16').Wrap(T('Notifications'), 'em').$CNotifications, UserUrl($User), 'MeButton FlyoutButton', array('title' => T('Notifications')));
+         echo Anchor(Sprite('SpNotifications', 'Sprite Sprite16').Wrap(T('Notifications'), 'em').$CNotifications, UserUrl($User), 'MeButton FlyoutButton NotificationsClear', array('title' => T('Notifications')));
```

Comment:
Sorry to nickle and dime you, but classes that are pure behaviour should be 'js-' prefixes. So something like `js-clear-notifications` is good. This is kind of a new thing, but will ensure that themers don't style on such classes.